modelci/hub/deployer/pytorch/utils.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum
```

Comment:
where do you use the DataType and type converter in PyTorch deployer?

Reply:
I made this change for compatibility with the new persistence module. While, it is beyond my consideration that this file was redundant in the first place. Shall we delete this file or just revert this change?